.github/workflows/release-native.yml

```diff
@@ -45,17 +44,23 @@ jobs:
       - run: sbt scala-native
         env:
           CI: true
+      - name: Zip artifact for deployment
+        if:  ${{ !startsWith(matrix.deploy.os, 'windows') }}
+        run: zip ${{ matrix.deploy.name }} ${{ env.BINARY_NAME }}
+      - name: Zip artifact for deployment
+        if:  ${{ startsWith(matrix.deploy.os, 'windows') }}
+        run: 7z a ${{ matrix.deploy.name }} ${{ env.BINARY_NAME }}
```

Comment:
one rule, perhaps:
```
  - name: Zip ...
    run: ${{
      startsWith(matrix.deploy.os, 'windows') &&
      7z matrix.deploy.name 'scalafmt.exe' ||
      zip matrix.deploy.name 'scalafmt'
    }}
```
and remove definition of `BINARY_NAME` under `env`.

Reply:
I couldn't make it work like this, but I removed `BINARY_NAME ` let me know what you think

Reply:
I tried that as well and unless there is something else to change I don't think that will work. It actually reported the previous line without any data.